### PR TITLE
fix: BX-1220

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-1220
Figma link (if any):

## What changed (plus any additional context for devs)

for empty wallets with no assets we're not fetching the native assets so this bug was happening

with this pr i'm introducing mock assets to have the native assets with prices coming from backend but balance being 0 so the user will see the tx fee correctly

## Screen recordings / screenshots

https://www.loom.com/share/85086669138243bdbcbcb1ba0415e8a3

### CODE REVIEW

File: src/entries/popup/hooks/useNativeAsset.ts [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
+}: {
+  chainId: ChainId;
+}): ParsedUserAsset | undefined | null => {
+  const nativeAssets = useNativeAssets();
+  const { chains } = useNetwork();
+  if (!nativeAssets) return null;
+  return chains
+    .map((chain) => {
+      const assetKey = `${NATIVE_ASSETS_MAP_PER_CHAIN[chain.id]}_${
+        ChainId.mainnet
```
Comment (greg-schrammel) [inline src/entries/popup/hooks/useNativeAsset.ts:27]: is mainnet right?

File: src/entries/popup/hooks/useNativeAsset.ts [hunk 2]
```diff
@@ -1,23 +1,52 @@
...
+        ...nativeAsset,
+        chainId: chain.id,
+        chainName: chainNameFromChainId(chain.id),
+        native: {
+          balance: { amount: '0', display: `0 ${nativeAsset?.symbol}` },
+        },
+        balance: { amount: '0', display: `0 ${nativeAsset?.symbol}` },
+      };
+    })
+    .find((c) => c.chainId === chainId);
```
Comment (greg-schrammel) [inline src/entries/popup/hooks/useNativeAsset.ts:40]: hm if we're filtering for the chain in the end is building the whole map even necessary?

File: src/entries/popup/hooks/useNativeAsset.ts (hunk 1 above)
Comment (estebanmino) [reply src/entries/popup/hooks/useNativeAsset.ts:27]: yes this is coming from here https://github.com/rainbow-me/browser-extension/blob/3d5e28eed26cc6e8d6c6ee4ec1b7e1e8bcd362eb/src/entries/popup/hooks/useNativeAssets.ts#L35C21-L35C21

File: src/entries/popup/hooks/useNativeAsset.ts (hunk 2 above)
Comment (estebanmino) [reply src/entries/popup/hooks/useNativeAsset.ts:40]: i'll do this at the beginning

**Review by brunobar79 — APPROVED** [review]
Good idea 🤝

**Review by greg-schrammel — APPROVED** [review]
👑

**Pull request merged**